extensions/blocks/podcast-player/templates/podcast-header.php

```diff
@@ -57,12 +60,35 @@
 	<div
 		id="<?php echo esc_attr( $player_id ); ?>__track-description"
 		class="jetpack-podcast-player__track-description"
+		<?php echo $is_amp ? '[text]="podcastPlayer.tracks[currentTrack].description"' : ''; ?>
 	>
 		<?php echo esc_html( $track['description'] ); ?>
 	</div>
 	<?php endif; ?>
 
 	<div class="jetpack-podcast-player__audio-player">
-		<div class="jetpack-podcast-player--audio-player-loading"></div>
+		<?php if ( $is_amp ) : ?>
+			<amp-audio
+				width="100%"
+				height="40"
+				artwork="<?php echo esc_url( $cover ); ?>"
+				artist="<?php echo esc_attr( $title ); ?>"
+				src="<?php echo esc_url( $track['src'] ); ?>"
+				title="<?php echo esc_attr( $track['title'] ); ?>"
+				[src]="podcastPlayer.tracks[currentTrack].src"
+				[title]="podcastPlayer.tracks[currentTrack].title"
+			>
+				<p fallback>
+					<a [href]="podcastPlayer.tracks[currentTrack].link" href="<?php echo esc_url( $track['link'] ); ?>">
+						<?php esc_html_e( 'Open episode page', 'jetpack' ); ?>
+					</a>
+					<a download [href]="podcastPlayer.tracks[currentTrack].src" href="<?php echo esc_url( $track['src'] ); ?>">
+						<?php esc_html_e( 'Download audio', 'jetpack' ); ?>
+					</a>
+				</p>
+			</amp-audio>
```

Comment:
I'd extract this to a dedicated AMP specific template rather than leaving inline.